utils/hash.ts

```diff
@@ -1,7 +1,16 @@
 import * as crypto from "crypto";
+import { FamilyMembers } from "../generated/definitions/FamilyMembers";
```

Comment:
should we use models?
```suggestion
import { FamilyMembers } from "../generated/models/FamilyMembers";
```

Reply:
Depends on how we use it, I've used definition because maybe we must check familyUID before generating and saving the new bonus

Reply:
i think it does not matter since the type is the same